pkg/auth/auth.go

```diff
@@ -165,6 +166,15 @@ func (u *UserAuth) isAllowed(verb string, itemsToCheck []resource) ([]Action, er
 		if err != nil {
 			return []Action{}, err
 		}
+		// If the "group" is versioned the user may be able to have access to any
```

Comment:
It does matter: If the user only have permissions to `apps/v1` won't be able to use `apps/v1beta1` but if it has permissions to use `apps` it can use both. The problem is that when it has permissions for `apps` and the request is made with `apps/v1` the method return "false" so we need to repeat the request with `apps`.

Reply:
Understood, thanks.